common/src/test/java/io/pravega/common/io/FixedByteArrayOutputStreamTests.java

```diff
@@ -23,8 +23,6 @@
 public class FixedByteArrayOutputStreamTests {
     /**
      * Tests that FixedByteArrayOutputStream works as advertised.
-     *
-     * @throws IOException
```

Comment:
I understand it doesn't matter for tests, but why would we be getting warnings for this @throws declaration?

Reply:
It's empty.